### ISSUE TITLE
Build errors on Android SDK 33

### ISSUE THREAD

**Issue by dotBeFoRE** [react-native-menu/menu#466]
When I try to build my app Expo throws this error when using Android SDK 33.
```
> Task :react-native-menu_menu:compileDebugKotlin
e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (28, 7): 'onLongPress' overrides nothing
e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (35, 7): 'onSingleTapConfirmed' overrides nothing 
e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (49, 35): Type mismatch: inferred type is MotionEvent? but MotionEvent was expected
e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (127, 15): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type Drawable?
e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (148, 15): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type Drawable?
e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (162, 45): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type SubMenu?
```
I know you don't support Expo, but it worked a couple of days ago, but now it stopped working. Maybe you might have an explanation.

**Comment by saravanakumargn** [comment]
> When I try to build my app Expo throws this error when using Android SDK 33.
> 
> ```
> > Task :react-native-menu_menu:compileDebugKotlin
> e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (28, 7): 'onLongPress' overrides nothing
> e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (35, 7): 'onSingleTapConfirmed' overrides nothing 
> e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (49, 35): Type mismatch: inferred type is MotionEvent? but MotionEvent was expected
> e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (127, 15): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type Drawable?
> e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (148, 15): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type Drawable?
> e: C:\Users\before\coach-twins-mobile-app\node_modules\@react-native-menu\menu\android\src\main\java\com\reactnativemenu\MenuView.kt: (162, 45): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type SubMenu?
> ```
> 
> I know you don't support Expo, but it worked a couple of days ago, but now it stopped working. Maybe you might have an explanation.

I am not using Expo. But I am getting the same error. It was working a few days back and it stopped working now.
```
/node_modules/@react-native-menu/menu/android/src/main/java/com/reactnativemenu/MenuView.kt: (162, 45): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type SubMenu?

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-menu_menu:compileDebugKotlin'.
> Compilation error. See log for more details
```

```
"@react-native-menu/menu": "^0.5.3", // i tried with 0.7.1 also. But, not working
```

**Comment by arvidnilber** [comment]
Created this patch for SDK 33 until a new version releases. It solved my issues, and the menu still works fine. 
https://gist.github.com/arvidnilber/478eefd1b44729184e818b2b7f0c7adb

**Comment by MatthewLamperski** [comment]
> Created this patch for SDK 33 until a new version releases. It solved my issues, and the menu still works fine. https://gist.github.com/arvidnilber/478eefd1b44729184e818b2b7f0c7adb

I love you.

**Comment by andreiwow2** [comment]
I also get
```
The Android Gradle plugin supports only kotlin-android-extensions Gradle plugin version 1.6.20 and higher.
The following dependencies do not satisfy the required version:
project ':react-native-menu_menu' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10
```

**Comment by Naturalclar** [comment]
Should be fixed in v0.7.3 🎉
